cli/src/commands/bookmark/list.rs

```diff
@@ -74,8 +78,10 @@ pub struct BookmarkListArgs {
     /// Show bookmarks whose local name matches
     ///
     /// By default, the specified name matches exactly. Use `glob:` prefix to
-    /// select bookmarks by wildcard pattern. For details, see
-    /// https://jj-vcs.github.io/jj/latest/revsets/#string-patterns.
+    /// select bookmarks by [wildcard pattern].
+    ///
+    /// [wildcard pattern]:
```

Comment:
(Non-blocking, I think?) Copying myself from Discord:

There's another issue that is OK now, but might become a problem later. For the links in https://jj-vcs.github.io/jj/prerelease/cli-reference/ to work, I bet we need each `[wildcard pattern]` in every help text to always link to the same URL. It will probably use the first URL from the first `[wildcard pattern]` for all of them.

MkDocs does not seem to warn when this rule is violated. Maybe it's something we could add to it.

Reply:
Here's a test gist: https://gist.github.com/ilyagr/1908ec6785a51df7c44d646bc430afb3 

MkDocs seems to act the same.

Reply:
we do need them to be the same, but I actually think that is a useful commonality for us to have.  The docs should echo themselves as much as possible.

Reply:
As I think about it, I become more worried, since as a human, it's hard for me to verify that this condition holds.

Could you create some sort of lint to check for it? It could be our own thing, or an MkDocs plugin, or a patch to MkDocs.

Reply:
I filed https://github.com/DavidAnson/markdownlint/issues/1475 but I don't have the cycles to implement it.

Reply:
It might be tricky to get markdownlint to work with MkDocs and `jj`'s docs. For one, MkDocs Markdown format predates CommonMark. For another, we'd need to find a good way to use Node.js tools in our dev environment and CI.

Reply:
How about this: we merge this for now. If, by the time this becomes a problem, there is no readily available solution to make it easy to maintain, we revert it. WDYT?

Reply:
works for me